docs/source/getting-started/index.rst

```diff
@@ -83,7 +83,7 @@ We can now run the app in two steps.
 You'll need two terminal windows running for this.
 In one terminal, at the root of the ``create-daml-app`` directory, run the command::
 
-    daml start
+    daml start --json-api-option=--leak-passwords-firesheep-style
```

Comment:
Should `daml start` imply this?

Reply:
Yes, `daml start` needs to work without passing extra options in a local development setup.

Reply:
Can we be confident nobody will use `daml start` in production? Is it possible to use the PostgreSQL backend with `daml start`?

If we're bypassing a security measure by default, I think we should be loud about it. Maybe print "insecure sandbox" instead of "sandbox" as the banner or something.

```
              .-') _   .-')      ('-.                         _  .-')     ('-.   
             ( OO ) ) ( OO ).  _(  OO)                       ( \( -O )  _(  OO)  
  ,-.-') ,--./ ,--,' (_)---\_)(,------.   .-----. ,--. ,--.   ,------. (,------. 
  |  |OO)|   \ |  |\ /    _ |  |  .---'  '  .--./ |  | |  |   |   /`. ' |  .---' 
  |  |  \|    \|  | )\  :` `.  |  |      |  |('-. |  | | .-') |  /  | | |  |     
  |  |(_/|  .     |/  '..`''.)(|  '--.  /_) |OO  )|  |_|( OO )|  |_.' |(|  '--.  
 ,|  |_.'|  |\    |  .-._)   \ |  .--'  ||  |`-'| |  | | `-' /|  .  '.' |  .--'  
(_|  |   |  | \   |  \       / |  `---.(_'  '--'\('  '-'(_.-' |  |\  \  |  `---. 
  `--'   `--'  `--'   `-----'  `------'   `-----'  `-----'    `--' '--' `------' 
  .-')     ('-.         .-') _  _ .-') _ .-. .-')              ) (`-.            
 ( OO ).  ( OO ).-.    ( OO ) )( (  OO) )\  ( OO )              ( OO ).          
(_)---\_) / . --. /,--./ ,--,'  \     .'_ ;-----.\  .-'),-----.(_/.  \_)-.       
/    _ |  | \-.  \ |   \ |  |\  ,`'--..._)| .-.  | ( OO'  .-.  '\  `.'  /        
\  :` `..-'-'  |  ||    \|  | ) |  |  \  '| '-' /_)/   |  | |  | \     /\        
 '..`''.)\| |_.'  ||  .     |/  |  |   ' || .-. `. \_) |  |\|  |  \   \ |        
.-._)   \ |  .-.  ||  |\    |   |  |   / :| |  \  |  \ |  | |  | .'    \_)       
\       / |  | |  ||  | \   |   |  '--'  /| '--'  /   `'  '-'  '/  .'.  \        
 `-----'  `--' `--'`--'  `--'   `-------' `------'      `-----''--'   '--'       
```

Alternatively, instead of making it a default at the `daml start` level, make it configurable through `daml.yaml` and add it to the default template? That would also give us the option of adding a comment with a link to more detailed docs or something. something like:

```
daml-start:
  # Conveniently disable TLS for local development.
  # Do not leave on for production use. See https://docs.daml.com/security.html.
  insecure: true
```

Reply:
I don’t want to tell users to have to add a whole bunch of very scary looking flags when they are just getting started and printing out scary looking banners goes in the same direction. Is there some precedence here in other tools that we could follow?

Reply:
Notwithstanding the outcome of this discussion, `daml start` implies the new option as of 185ea518a2, with the above doc change reverted to just `daml start`.